test/unit/lib/plugins/aws/package/compile/events/cloud-front.test.js

```diff
@@ -1573,88 +1122,278 @@ describe('test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js',
-      // Replaces
-      // https://github.com/serverless/serverless/blob/85e480b5771d5deeb45ae5eb586723c26cf61a90/lib/plugins/aws/package/compile/events/cloudFront/index.test.js#L169-L204
-
+    it('should throw if function `timeout` is greater than 5 for for `functions[].events.cloudfront.evenType: "viewer-request"', async () => {
       return expect(
-        runServerless({ fixture: 'function', command: 'package' })
-      ).to.eventually.be.rejected.and.have.property('code', 'TODO');
+        runServerless({
+          fixture: 'function',
+          command: 'package',
+          configExt: {
+            functions: {
+              first: {
+                timeout: 6,
+                handler: 'index.handler',
+                events: [
+                  {
+                    cloudFront: {
+                      eventType: 'viewer-request',
+                      origin: 's3://bucketname.s3.amazonaws.com/files',
+                    },
+                  },
+                ],
+              },
+            },
+          },
+        })
+      ).to.eventually.be.rejected.and.have.property(
+        'code',
+        'LAMBDA_EDGE_UNSUPPORTED_TIMEOUT_VALUE'
+      );
     });
 
-    it.skip('TODO: should throw if function `memorySize` is greater than 3008 for `functions[].events.cloudfront.evenType: "origin-request"`', async () => {
+    it.skip('should throw if function `memorySize` is greater than 10240 for `functions[].events.cloudfront.evenType: "origin-request"`', async () => {
+      // TODO seems like max memory limit is changes to 10240
+      // NOTE: this config is failed to validate.
       // Replaces partially
       // https://github.com/serverless/serverless/blob/85e480b5771d5deeb45ae5eb586723c26cf61a90/lib/plugins/aws/package/compile/events/cloudFront/index.test.js#L206-L242
 
       return expect(
-        runServerless({ fixture: 'function', command: 'package' })
-      ).to.eventually.be.rejected.and.have.property('code', 'TODO');
+        runServerless({
+          fixture: 'function',
+          command: 'package',
+          configExt: {
+            functions: {
+              basic: {
+                memorySize: 10241,
+                events: [
+                  {
+                    cloudFront: {
+                      eventType: 'origin-request',
+                      origin: 's3://bucketname.s3.amazonaws.com/files',
+                    },
+                  },
+                ],
+              },
+            },
+          },
+        })
+      ).to.eventually.be.rejected.and.have.property('code', 'LAMBDA_EDGE_UNSUPPORTED_MEMORY_SIZE');
     });
 
-    it.skip('TODO: should throw if function `memorySize` is greater than 3008 for `functions[].events.cloudfront.evenType: "origin-response"`', async () => {
+    it.skip('should throw if function `memorySize` is greater than 10240 for `functions[].events.cloudfront.evenType: "origin-response"`', async () => {
```

Comment:
Should be unskipped

Reply:
https://github.com/serverless/serverless/issues/8528#issuecomment-1383959217
same config error

Reply:
Same here